app/ts/components/simulationExplaining/customExplainers/SimpleTokenApprovalVisualisation.tsx

```diff
@@ -30,7 +30,7 @@ export function SimpleTokenApprovalVisualisation(params: SimpleTokenApprovalVisu
 		<div class = 'box' style = 'background-color: var(--alpha-005); box-shadow: unset; margin-bottom: 0px;'>
 			<span class = 'log-table' style = 'justify-content: center; column-gap: 5px;'>
 				<div class = 'log-cell' style = 'justify-content: right;'>
-					{ params.approval.type !== 'Token' ?
+					{ params.approval.type !== 'Erc20Token' ?
```

Comment:
Possibly going to be other non-erc20 assets than 721?

Reply:
yeah, this is work towards adding erc1155